### PR TITLE
New record

### LINKED ISSUE #99: Adding new records.
This isn't quite so straightforward.

* Creating new `records` are dependent on related `price_guide` prices.
* You need to search the price guide, find the record.
* Create new record based on price guide

### PR DESCRIPTION
**Pull request by tchorzewski1991**
Fix #99 

show_price

new_record

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -39,8 +39,9 @@ gem 'blazer'
 gem 'ruby-mp3info', '~> 0.8.4'
 gem 'ken', '~> 0.2.1'
 gem 'randumb', '~> 0.5.0'
-gem 'mini_magick', '~> 4.0.2'
-gem 'refile', '~> 0.5.3', :require => ['refile/rails', 'refile/image_processing']
+gem "refile", require: "refile/rails"
```
Comment (holden) [inline Gemfile]: Never leave gems unversioned if the app depends on them (things like rollbar, or mini magick perhaps are fine) but refile and devise when being updated to major version will break everything.

File: app/helpers/price_helper.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module PriceHelper
+  def price_header price
+    "#{price.artist_name}, #{price.label_name}"
+  end
```
Comment (holden) [inline app/helpers/price_helper.rb]: This one is unnecessary.  You'll forget what the helper does in a month and have to check it again anyway.

Comment (holden) [inline app/helpers/price_helper.rb]: `price_header` the other 2 are fine.

File: app/models/price.rb [hunk 3]
```diff
@@ -4,19 +4,11 @@ class Price < ActiveRecord::Base
   belongs_to :artist
   belongs_to :label
   belongs_to :user
-  has_one :record
+  # has_one :record ????
```
Comment (holden) [inline app/models/price.rb]: weird.

**Comment by tchorzewski1991** [comment]
I fixed that stauff that you are asking for :)

File: Gemfile [hunk 4]
```diff
@@ -41,6 +41,7 @@ gem 'ken', '~> 0.2.1'
 gem 'randumb', '~> 0.5.0'
 gem 'mini_magick', '~> 4.0.2'
 gem 'refile', '~> 0.5.3', :require => ['refile/rails', 'refile/image_processing']
+gem "cocoon"
```
Comment (holden) [inline Gemfile]: cocoon too!

File: app/models/price.rb (hunk 3 above)
Comment (holden) [inline app/models/price.rb]: Just wack it.  It's very wrong in so many ways.

**Pull request merged**